server/portal/apps/accounts/migrations/0005_auto_20201112_1954.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-11-12 19:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0004_auto_20200318_2055'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='notificationpreferences',
+            name='announcements',
+            field=models.BooleanField(default=True, verbose_name='Receive occasional announcements from sal.frontera'),
```

Comment:
This migration shouldn't be committed. We should also figure out why it keeps getting generated.